ern-local-cli/src/index.ts

```diff
@@ -55,21 +57,36 @@ function showInfo() {
 }
 
 function showVersion() {
-  // get electrode-native local-cli version
-  if (config.get('platformVersion')) {
-    log.info(`ern-local-cli : ${config.get('platformVersion')}`)
-  }
-  // get electrode-native global-cli version
-  const packageInfo = JSON.parse(
-    execSync('npm ls -g electrode-native --json').toString()
-  )
-  if (packageInfo && packageInfo.dependencies) {
-    log.info(
-      `electrode-native : ${
-        packageInfo.dependencies['electrode-native'].version
-      }`
+  let packageInfo
+  try {
+    const pkg = path.join(
+      execSync('yarn global dir')
+        .toString()
+        .trim(),
+      'node_modules',
+      'electrode-native',
+      'package.json'
     )
+    packageInfo = fs.existsSync(pkg) ? require(pkg) : ''
+  } catch (err) {
+    log.trace(err)
   }
+
+  if (!packageInfo) {
+    try {
+      packageInfo = JSON.parse(
+        execSync(`npm ls -g electrode-native --json`).toString()
+      ).dependencies['electrode-native']
+    } catch (err) {
+      log.trace(err)
+    }
+  }
+
+  const globalVersion = packageInfo ? packageInfo.version : ''
+  log.info(`electrode-native: ${globalVersion ? globalVersion : 'unknown'}`)
```

Comment:
minor: can we use the consistent notation like https://github.com/electrode-io/electrode-native/pull/1448/files#diff-7c73337d65768147975cba961c7b6680R48

Reply:
What exactly do you mean? Replace `unknown` with `-NONE-`?

Reply:
We can have -UNKNOWN-, no reservations with this :)

Reply:
OK I changed it to `-UNKNOWN-`